typings/result.d.ts

```diff
@@ -0,0 +1,126 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+declare global {
+  module LH {
+    /**
+     * The full output of a Lighthouse run.
+     */
+    export interface Result {
+      /**
+       * The URL that was supplied to Lighthouse and initially navigated to.
+       */
+      initialUrl: string;
+      /**
+       * The post-redirects URL that Lighthouse loaded.
+       */
+      url: string;
+      /**
+       * The ISO-8601 timestamp of when the results were generated.
+       */
+      fetchedAt: string;
+      /**
+       * The version of Lighthouse with which these results were generated.
+       */
+      lighthouseVersion: string;
+      /**
+       * An object containing the results of the audits, keyed by the audits' `id` identifier.
+       */
+      audits: Record<string, Audit.Result>;
+      /**
+       * The top-level categories, their overall scores, and member audits.
+       */
+      reportCategories?: Result.Category[];
+      /**
+       * Descriptions of the groups referenced by 
+       */
+      reportGroups?: Record<string, Result.Group>;
+      /**
+       * Description of the runtime configuration used for gathering these results.
+       */
+      runtimeConfig: Result.RuntimeConfig;
```

Comment:
lets add a bit more whitespace (and even a comment?) to separate lhr-non-lite

Reply:
done